src/frontend/src/app/components/layout/header/header.component.html

```diff
@@ -45,7 +45,7 @@
                   <button mat-menu-item [routerLink]="['/events/add']">Add Event</button>
                 </mat-menu>
 
-                <button mat-icon-button [matMenuTriggerFor]="addMenu">
+                <button *ngIf="userType === 2" mat-icon-button [matMenuTriggerFor]="addMenu">
```

Comment:
The ngIf will show the button only when the condition is met.  In this case, the when the userType is equal to 2 which is a parent.  Should this be a 3 (educator) instead?

This brings up another issue. The educator should not be able to create an event before they are approved.